src/aleph/netty.clj

```diff
@@ -1271,6 +1330,13 @@ initialize an DnsAddressResolverGroup instance.
          ssl-context (when (some? ssl-context)
                        (coerce-ssl-server-context ssl-context))
 
+         pipeline-builder
+         (if (some? chan-group)
+           (fn [^ChannelPipeline p]
+             (append-handler-to-pipeline p "group-tracker" (channel-tracking-handler chan-group))
+             (pipeline-builder p))
+           pipeline-builder)
+
          pipeline-builder
          (if ssl-context
            (fn [^ChannelPipeline p]
```

Comment:
For future reference, `cond->` would be the better way to do this, since we can combine them.